_posts/2021-07-05-newsletter.md

```diff
@@ -0,0 +1,199 @@
+
+Correspondingly, over the course of the 0.5 development cycle, there has been an explosion in the number of Lua plugins, ranging from rewrites of popular Vimscript plugins to completely new ones that would not have been possible in Vimscript -- often from contributors who were completely new to (neo)vim plugin development and were averse to learning Vimscript for that task. As a positive side effect of the long development cycle, many of these are already fully-featured and stable at the time of the 0.5 release!
+
+Here is a small and non-representative list of Lua plugins:
+
+- [Plenary](https://github.com/nvim-lua/plenary.nvim) -- A lua library of useful utilities for developing Neovim plugins (some of which will later be integrated into core).
+- [Packer](https://github.com/wbthomason/packer.nvim) -- A use-package like plugin manager with support for plugin dependencies, lazy-loading, and installing luarocks.
+- [Telescope](https://github.com/nvim-telescope/telescope.nvim) -- A highly extendable fuzzy finder over lists.
+- [Gitsigns](https://github.com/lewis6991/gitsigns.nvim) -- A Lua (teal) plugin for showing and interacting with changes on files a git repository (asynchronously).
+- [Nvim-compe](https://github.com/nvim-lua/completion-nvim) -- An auto completion framework for various sources, including Neovim's builtin LSP client.
+- [Nvim-dap](https://github.com/mfussenegger/nvim-dap) -- A debug adapter protocol implementation for step-through debugging of your code.
+- [Colorizer](https://github.com/norcalli/nvim-colorizer.lua) -- A high-performance color highlighter for Neovim without any external dependencies.
+- [Formatter](https://github.com/mhartington/formatter.nvim) -- A plugin for asynchronously executing external formatting tools on the current buffer or range.
+- [Hop.nvim](https://github.com/phaazon/hop.nvim) - Easy-motion without messing with your buffer.
+- [Neogit](https://github.com/TimUntersberger/neogit) - Magit but for Neovim.
+
+Not all of these are actually written in Lua: there are many other languages (some of which are typed) that compile to Lua, e.g.,
+- [Fennel(Lisp)](https://fennel-lang.org) via [Aniseed],(https://github.com/Olical/aniseed) or [Fennel-nvim](https://github.com/jaawerth/fennel-nvim),
+- [Teal](https://github.com/teal-language/tl) (a typed dialect of Lua),
+- [Amulet ML](https://amulet.works),
+- [MoonScript](https://github.com/leafo/moonscript) via [nvim-moonmaker](https://github.com/svermeulen/nvim-moonmaker) ,
+- [Vim9script](https://github.com/tjdevries/vim9jit) (you read that right).
```

Comment:
@tjdevries is that something we can advertise already xD ?

Reply:
at least it should be clear that it is quite experimental compared to the earlier items in the same list  :]

Reply:
@tjdevries let me know how you want this to be referenced -- if at all!

Reply:
Feel free to reference it, but just say "This is an experiment to determine if it's possible for neovim to support vim9script" or something like that. It's a fun project :)

Reply:
the current comment is "(as a proof of concept)" -- github doesn't update these conversations...